<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>790</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Input Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Insert the DEM file of the Study Region</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Insert the Proximity to Waterbodies of the Study Region</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Insert the Rainfall Interpolation file of the Study Region</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>AHP</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>580</width>
          <height>180</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>Elevation</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Slope</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Proximity</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Rainfall</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Elevation</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Slope</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Proximity </string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rainfall</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0.80</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>0.57</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0.80</string>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string>0.57</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>1.25</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>1.25</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="2" column="3">
         <property name="text">
          <string>0.71</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>1.75</string>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string>1.75</string>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string>1.40</string>
         </property>
        </item>
        <item row="3" column="3">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Assign weights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Elevation</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Slope</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Proximity to WaterBodies</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="value">
           <double>0.250000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Rainfall </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
          <property name="value">
           <double>0.350000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
